src/sanitize-argument.ts

```diff
@@ -35,8 +35,12 @@ export async function sanitize_all(argv: any) {
     mwActionApiPath,
     mwRestApiPath,
     mwModulePath,
+    insecure,
   } = argv
 
+  // set insecure to false by default
```

Comment:
This is quite clear, you don't need to put a comment here

Reply:
There is nothing to sanitize here. Unfortunately there is not a real place defined looks like for such an initialisation, but I would say either in `src/sanitize-argument.ts` or in Downloader constructor (like `--webp`)

Reply:
directly in Downloader constructor works